packages/components/src/ButtonGroup/ButtonGroup.Context.js

```diff
@@ -1,4 +1,7 @@
 import { createContext, useContext } from 'react';
 
-export const ButtonGroupContext = createContext({});
+/** @type {import('./types').ButtonGroupContext} */
+const defaultContext = {};
+
+export const ButtonGroupContext = createContext(defaultContext);
```

Comment:
This is one way of typing a context, where you split the default context and cast it. The other was is below where you just type the return of createContext and TS will infer the type of the passed in default context.

Reply:
I see! I'm cool with this 👍